components.rst

```diff
@@ -0,0 +1,35 @@
+Components
+==========
+The library consists of three main parts.
+
+TaskScheduler
+-------------
+The ``TaskScheduler`` stores tasks and schedules Executions. Foreach tasks there
+exists always on execution in the status ``planned``, ``running``. If the task
+will only one time it only get one execution at all. For recurring tasks the
+execution will always generated for the next run time.
```

Comment:
`For recurring tasks a new execution will be generated when the old one is completed.`?